frontend/coprs_frontend/coprs/logic/complex_logic.py

```diff
@@ -489,6 +488,11 @@ def generate_build_config(cls, copr, chroot_id):
         repos.extend(cls.get_additional_repo_views(copr.repos_list, chroot_id))
         repos.extend(cls.get_additional_repo_views(chroot.repos_list, chroot_id))
 
+        repo_priority = copr.repo_priority
+        if repo_priority is not None:
+            for repo in repos:
+                repo["priority"] = repo_priority
+
```

Comment:
I am not sure if we want to set the priority only for `copr_base` or `copr_base_devel` as well. But IMHO this code block needs to be at least above the 

```python
repos.extend(cls.get_additional_repo_views(copr.repos_list, chroot_id))
repos.extend(cls.get_additional_repo_views(chroot.repos_list, chroot_id))
```

@praiskup, you might want check this, if I am understanding the feature correctly.

Reply:
I agree, `copr_base` and `copr_base_devel` only, and moving up.